lib/core/crypto/eosdart/src/client.dart

```diff
@@ -3,7 +3,7 @@ import 'dart:convert';
 import 'dart:typed_data';
```

Comment:
These changes need to be merged into main

This is legacy code, and pretty horrible design to return dynamic with two totally different classes for no reason at all. 

Split method in 2, now it's much better

Also there's changes because dio.Response is different from the previous response object, and we need to access response.data for the JSON that was returned. 

This caused an Result.error to be returned even though the transaction was successful. So this is an actual bugfix.